### ISSUE TITLE
Script error while removing an item from search list

### ISSUE THREAD

**Issue by b-jesch** [fivebanger/kodi_addons#4]
It's not possible to delete an item from the search list.

```
2024-07-25 14:31:56.992 T:42101   error <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'KeyError'>
                                                   Error Contents: 'list'
                                                   Traceback (most recent call last):
                                                     File "/home/jesch/.kodi/addons/plugin.audio.radiode/addon.py", line 15, in <module>
                                                       dispatcher.route()
                                                     File "/home/jesch/.kodi/addons/plugin.audio.radiode/addon_utils.py", line 139, in route
                                                       funtion(data)
                                                     File "/home/jesch/.kodi/addons/plugin.audio.radiode/src/radio_de.py", line 235, in remove_from_serach
                                                       search_list = getattr(self, data['list'])
                                                   KeyError: 'list'
                                                   -->End of Python script error report<--
                                                   
```

**Comment by fivebanger** [comment]
Confirmed: "Remove from search" for stations search and podcasts search does not work in v1.1.2.

**Comment by fivebanger** [comment]
Fixed in v1.1.3

**Comment by b-jesch** [comment]
Just installed v1.1.3. Works as expected now. Thank You.
